src/pages/productDetail/seulgi.scss

```
@@ -0,0 +1,80 @@
+@import '../../styles/variables.scss';
+
+.productDetail {
+  display: flex;
+  flex-direction: row;
+  width: 100vw;
+
+  h2 {
+    font-size: 22px;
+    margin-bottom: 50px;
+  }
+
+  section {
+    background-color: white;
+
+    img {
+      width: 100%;
+    }
+  }
+
+  ul {
+    font-size: 12px;
+
+    li {
+      margin-bottom: 10px;
+    }
+
+    .description {
+      font-size: 12px;
+      font-weight: 300;
+      color: #666;
+      line-height: 20px;
+      margin-bottom: 30px;
+    }
+    .price {
+      font-weight: 600;
+    }
+  }
+
+  .leftSide {
+    width: 55%;
+    min-width: 500px;
+  }
+
+  .rightSide {
+    width: 45%;
+    right: 0;
+    .sideDetail {
+      position: fixed;
+      width: 750px;
+      padding: 120px 10%;
+    }
```

```suggestion
  .rightSide {
    width: 45%;
    right: 0;
    
    .sideDetail {
      position: fixed;
      width: 750px;
      padding: 120px 10%;
    }
```
다른 셀렉터들끼리는 공백 부여해주세요!

---

해당 부분은 productDetail 브랜치에 수정하여 올렸습니다!